Assets/ModScripts/Submodules/SimonSkips.cs

```diff
@@ -6,16 +6,13 @@
 using KModkit;
 using Random = UnityEngine.Random;
 
-
-
 public class SimonSkips : Puzzle
 {
     int[] arrowColours;
-    readonly int[] orderedArrows;
-    readonly List<int> finalSequence = new List<int>(); // List of indexes to press
-    readonly List<int> inputtedSequence = new List<int>();
-    readonly bool submitEmpty = false;
-    readonly int inputNum = 0;
+    int[] orderedArrows;
+    List<int> finalSequence = new List<int>(); // List of indexes to press
+    List<int> inputtedSequence = new List<int>();
+    bool submitEmpty = false;
```

Comment:
Uhhh I think you forgot some readonlys lol

Reply:
Why should these be readonly if I change them?